language/tools/move-cli/tests/sandbox_tests/generate_struct_layout/args.exp

```diff
@@ -5,16 +5,6 @@ Command `sandbox generate struct-layouts --module storage/0x00000000000000000000
   STRUCT:
     - t: U64
     - b: BOOL
-AccountAddress:
```

Comment:
Could you explain the test cases changes? I don't think I understand the code enough to understand the change